sources/api/netdog/src/networkd/config/netdev.rs

```diff
@@ -144,4 +147,175 @@ impl NetDevConfig {
 
         Ok(path)
     }
+
+    // The following *mut() methods are private and primarily meant for use by the NetDevBuilder.
```

Comment:
nit, another case where I would use doc comments even though it is private. not sure if we have a consistent style on this.

Reply:
> not sure if we have a consistent style on this.

I believe we've typically followed this pattern of doc comments on "public" symbols/functions, and regular comments on "private" symbols and things.